core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/MarkersExtractor.kt

```diff
@@ -54,7 +55,8 @@ internal object MarkersExtractor {
 
     private fun getFields(markerClass: KClass<*>, nullableProperties: Boolean): List<GeneratedField> {
         val order = getPropertyOrderFromPrimaryConstructor(markerClass) ?: emptyMap()
-        return markerClass.memberProperties.sortedBy { order[it.name] ?: Int.MAX_VALUE }.mapIndexed { _, it ->
+        val structuralProperties = markerClass.memberProperties.filter { it.findAnnotation<ScopeProperty>() == null }
```

Comment:
you could probably just use .hasAnnotation<>() :)

Reply:
Didn't know, thanks!